pkgs/development/python-modules/pyjwt/default.nix

```diff
@@ -8,13 +8,13 @@
 
 buildPythonPackage rec {
   pname = "pyjwt";
-  version = "2.4.0";
+  version = "2.5.0";
   disabled = pythonOlder "3.6";
```

Comment:
Nit: This can be updated to "3.7" now, per https://github.com/jpadilla/pyjwt/pull/777.